### ISSUE TITLE
[Question]: how can I set learning rate

### ISSUE THREAD

**Issue by Tianyue1008** [run-llama/llama_index#19616]
### Question Validation

- [x] I have searched both the documentation and discord for an answer.

### Question

I want to fine tuning the embedding model, but I can set the learning rate. Where can I find this parameter.

**Comment by Tianyue1008** [comment]
how can I use my local embedding model @dosu

**Comment by Tianyue1008** [comment]
I want to fine tuning my local embedding model. I use SentenceTransformersFinetuneEngine but it can't set the learning_rate, so I use EmbeddingAdapterFinetuneEngine. But I can't pass my local embedding model in to EmbeddingAdapterFinetuneEngine. @dosu

**Comment by Tianyue1008** [comment]
ok, I can use the SentenceTransformersFinetuneEngine, but I want to know it's default learning_rate. @dosu
